src/antares/model/study.py

```diff
@@ -285,6 +302,24 @@ def delete_binding_constraint(self, constraint: BindingConstraint) -> None:
     def delete(self, children: bool = False) -> None:
         self._study_service.delete(children)
 
+    def create_variant(self, variant_name: str) -> "Study":
+        """
+        Creates a new variant for the study
+
+        Args:
+            variant_name: the name of the new variant
+        Returns: The variant in the form of a Study object
+        """
+        variant_id = self._study_service.create_variant(variant_name)
+        config = self._study_service.config
+
+        if isinstance(config, APIconf):
```

Comment:
All of this should be handled inside the API layer inside the create_variant method as we don't want to put API or local related things inside the study model